libs/utils/tradeHistoryAPI.ts

```diff
@@ -2,41 +2,54 @@ import { ARBITRUM, ARBITRUM_RINKEBY } from '@libs/constants';
 import BigNumber from 'bignumber.js';
 
 // Base API URL
-const BASE_TRADE_HISTORY_API = process.env.NEXT_PUBLIC_TRADE_HISTORY_API;
+// const BASE_TRADE_HISTORY_API = process.env.NEXT_PUBLIC_TRADE_HISTORY_API;
+const BASE_TRADE_HISTORY_API = 'https://dev.api.tracer.finance/poolsv2/tradeHistory';
 
 type NetworkType = typeof ARBITRUM_RINKEBY | typeof ARBITRUM;
 
-type CommitType = 'LongBurn' | 'LongMint' | 'ShortMint' | 'ShortBurn';
+export const COMMIT_TYPES_V2 = {
+    LONG_MINT: 'LongMint',
+    LONG_BURN: 'LongBurn',
+    SHORT_MINT: 'ShortMint',
+    SHORT_BURN: 'ShortBurn',
+    LONG_BURN_SHORT_MINT: 'LongBurnShortMint',
+    SHORT_BURN_LONG_MINT: 'ShortBurnLongMint',
+} as const;
 
 // Raw API return types
 type Result = {
-    collateralAmount: string;
-    collateralSymbol: string;
     date: number;
-    tokenAddress: string;
-    tokenAmount: string;
-    tokenName: string;
-    tokenPrice: string;
-    tokenSymbol: string;
-    type: CommitType;
-    userAddress: string;
+    type: typeof COMMIT_TYPES_V2[keyof typeof COMMIT_TYPES_V2];
     tokenDecimals: number;
+    tokenInAddress: string;
+    tokenInSymbol: string;
+    tokenInName: string;
+    tokenInAmount: string;
+    price: string;
+    fee: string;
+    tokenOutAddress: string;
+    tokenOutSymbol: string;
+    tokenOutName: string;
+    tokenOutAmount: string;
```

Comment:
this type has been updated to include the following fields:

`priceTokenAddress`: string
`priceTokenName`: string
`priceTokenSymbol`: string